### DISCUSSION TITLE
View count & hamBurgerMenu for mobile devices

### DISCUSSION THREAD

**Discussion by Astrodevil** [adityatelange/hugo-PaperMod#1132] [Q&A] [answered]
Is there any way to show view counts or total reads on blogs or pages in this theme?
Reading time is good but views are also required, Much Needed

Menu button is also needed for mobile devices, Is it available or not? I tried editing `config` with `hamBurgerMenu: true` It didn't work

**Comment by jesse-wei** [comment, marked as answer]
See https://github.com/adityatelange/hugo-PaperMod/discussions/943.
